apps/meteor/app/api/server/v1/oauthapps.ts

```diff
@@ -43,6 +43,30 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'oauth-apps/:_id',
+	{
+		authRequired: true,
+		validateParams: { POST: isUpdateOAuthAppParams },
+	},
+	{
+		async delete() {
```

Comment:
Although this is correct, that's is not following the standards of our REST API, as briefly described in our docs: https://developer.rocket.chat/rocket.chat/contribute-to-rocket.chat/code-style-guide#standards

So for `v1` we should stick to `/v1/namespace.action`, which in this case would like `oauth-apps.create` (which already exists btw), `oauth-apps.update` and `oauth-apps.delete`